plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/notation/crowsfoot/CrowsFootDiagramNotation.java

```diff
@@ -115,6 +114,27 @@ private void createTargetDecorator(PolylineConnection conn, Color bckColor, Colo
         conn.setTargetDecoration(targetDecor);
     }
 
+    private List<DBSTableIndex> getIndexSourceAttributes(
+        Collection<? extends DBSTableIndex> indexes,
+        List<ERDEntityAttribute> sourceAttr,
+        DBRProgressMonitor monitor) throws DBException {
```

Comment:
monitor must be the fist parameter. Also please use NotNull annotations

Reply:
done